spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -29,65 +29,22 @@ def create_workspace_csv(workspaces, client_id_column)
   temp_file
 end
 
-shared_context 'release brick' do
-  before(:all) do
-    config_path = ConfigurationHelper.create_interpolated_tempfile(
-      @config_template_path,
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)